tests/Doctrine/Tests/DBAL/Types/BinaryTest.php

```diff
@@ -17,7 +17,7 @@ class BinaryTest extends \Doctrine\Tests\DbalTestCase
     protected $platform;
 
     /**
-     * @var \Doctrine\DBAL\Types\BinaryType
+     * @var \Doctrine\DBAL\Types\Type
```

Comment:
It would make more sense to instantiate the object via `new` instead of the factory and therefore having to loosen the type. Otherwise, we're testing the factory and the type.